src/rdds/exploration_rankscore/assemble_unique_variant_id.py

```diff
@@ -0,0 +1,46 @@
+import pandas as pd
+from typing import List
+
+
+def assemble_unique_variant_id(pos: int,
+                               id: bytes,
+                               alt: bytes,
+                               ref: bytes) -> bytes:
+    """
```

Comment:
```suggestion
def assemble_unique_variant_id(chromosome_pos: int,
                               variant_id: bytes,
                               alt_allele: bytes,
                               ref_allele: bytes) -> bytes:
    """
```
I know this, but to someone new, but I think it  might be good for someone not familiar with the VCF format to be more explicit